src/components/TransactionFailText/index.tsx

```diff
@@ -35,27 +42,17 @@ export const TransactionFailText = ({
   estimationStatus,
 }: TransactionFailTextProps): React.ReactElement | null => {
   const classes = useStyles()
-  const threshold = useSelector(currentSafeThreshold)
   const isWrongChain = useSelector(shouldSwitchWalletChain)
   const isGranted = useSelector(grantedSelector)
 
-  if (estimationStatus !== EstimationStatus.FAILURE && !(isCreation && !isGranted)) {
+  if ((estimationStatus !== EstimationStatus.FAILURE || !isExecution) && !(isCreation && !isGranted)) {
```

Comment:
When is the last part of this condition needed? I would assume only owners can create transactions but it seems confusing.

Reply:
It may be worthwhile adding a commet here to clarify this clause.

Reply:
Only owners (isGranted = true) can create transactions, yes.

Maybe if I reverse the condition it would look more legible? E.g.

```
const showError = (isExecution && estimationStatus === EstimationStatus.FAILURE &&) || (isCreation && !isGranted)
if (! showError) return null
```

WDYT?

Reply:
That's a lot clearer. Is there a specific reason we use `isGranted` as opposed to `isOwner`? It makes sense either way, so no need to change.

Reply:
I think its a lot clearer this way, thanks! I am still not sure if the last part could be removed since if `isCreation` is true, `isGranted` should also be true so the whole expression would always be false